go/teams/teams.go

```diff
@@ -298,6 +298,27 @@ func (t *Team) MemberRole(ctx context.Context, uv keybase1.UserVersion) (keybase
 	return t.chain().GetUserRole(uv)
 }
 
+func (t *Team) WasMostRecentlyAddedByInvitelink(uv keybase1.UserVersion) bool {
+	chain := t.chain().inner
+	logPoints, ok := chain.UserLog[uv]
+	if !ok {
+		return false
+	}
+	latestLogPointAddedAtIdx := len(logPoints) - 1
+	for _, usedInvites := range chain.UsedInvites {
+		for _, usedInvite := range usedInvites {
+			if usedInvite.Uv == uv {
+				invitelinkLogPointAddedAtIdx := usedInvite.LogPoint
+				if invitelinkLogPointAddedAtIdx == latestLogPointAddedAtIdx {
```

Comment:
this should check the category but it might not be in active invites anymore. depends on where we put it, maybe we can have a helper id->invite that works after cancelling/expiring

Reply:
why not
```
if usedInvite.LogPoint == latestLogPointAddedAtIdx {
```
?
temp variable is not making the statement shorter

Reply:
It was mainly for clarity/explictness about what the variables are

Reply:
kk LGTM